web/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM node:20-slim AS build
+
+WORKDIR /app
+
+COPY package.json pnpm-lock.yaml ./
+RUN corepack enable && corepack prepare pnpm@latest --activate
+RUN pnpm install --frozen-lockfile
+
+COPY . .
+
+ARG APP_API_HOST
+ARG APP_GITHUB_APP_NAME
+ENV APP_API_HOST=${APP_API_HOST}
+ENV APP_GITHUB_APP_NAME=${APP_GITHUB_APP_NAME}
+
+RUN pnpm build
+
+FROM nginx:alpine AS runtime
```

Comment:
for local setup I would prefer having a node runtime instead of nginx in order to provide devtools